src/containerapp/azext_containerapp/custom.py

```diff
@@ -3807,3 +3807,129 @@ def remove_revision_label(cmd, resource_group_name, name, label, no_wait=False):
         return r['properties']['configuration']['ingress']['traffic']
     except Exception as e:
         handle_raw_exception(e)
+
+
+def show_environment_premium_ingress(cmd, name, resource_group_name):
+    _validate_subscription_registered(cmd, CONTAINER_APPS_RP)
+
+    try:
+        env = ManagedEnvironmentPreviewClient.show(cmd, resource_group_name, name)
+        if not env:
+            raise ResourceNotFoundError(f"The containerapp environment '{name}' does not exist")
```

Comment:
ni: the `if not env` is not works when resource group not exists or environment not exists
 az containerapp env premium-ingress show -n a -g b
Command group 'containerapp env premium-ingress' is in preview and under development. Reference and support levels: https://aka.ms/CLI_refstatus
**(ResourceGroupNotFound) Resource group 'b' could not be found.**


az containerapp env premium-ingress show -n a -g arcenv-xinyu1-rg
Command group 'containerapp env premium-ingress' is in preview and under development. Reference and support levels: https://aka.ms/CLI_refstatus
**(ResourceNotFound) The Resource 'Microsoft.App/managedEnvironments/a' under resource group 'arcenv-xinyu1-rg' was not found. For more details please go to https://aka.ms/ARMResourceNotFoundFix**